File: other/deepseek/TheLargestElementInArray.java

public class TheLargestElementInArray {
    

    private static int largestElement(int[] arr) {

        if (arr.length < 1 || arr == null) {
            throw new IllegalArgumentException("The array must contain at least 1 number");
        }

        int biggestNumber = arr[0];

        for (int i : arr) {
            if (i > biggestNumber) biggestNumber = i;
        }
        return biggestNumber;
    }

    public static void main(String[] args){

        int[] array = new int[] {1,2,546,4,513,78};
        System.out.println("Number: " + largestElement(array));
    }
}

File: other/deepseek/TheLargestElementInArray.js

const largestElement = (arr) => {

    let biggestNumber = arr[0];

    for (let i = 1; i < arr.length; i++){

        if (arr[i] > biggestNumber) biggestNumber = arr[i];
    }

    return biggestNumber;
}

const arr = [1,2,3,34,5,45,21,3,8];
console.log(largestElement(arr));